tests/context_tests.rs

```diff
@@ -103,7 +104,14 @@ fn create_public_sealed_object() -> tss_esapi::tss2_esys::TPM2B_PUBLIC {
 }
 
 fn create_ctx_without_session() -> Context {
-    unsafe { Context::new(Tcti::Mssim(Default::default())).unwrap() }
+    let tcti = match env::var("TEST_TCTI") {
+        Err(_) => Tcti::Mssim(Default::default()),
+        Ok(tpe) => match &tpe[..] {
```

Comment:
I think it would be excellent to use the `FromStr` trait on `Tcti` here! The implementation is [here](https://github.com/parallaxsecond/rust-tss-esapi/blob/d55f3fca59d1c2ac40d49ff371732440c8b1c2f8/src/tcti.rs#L71), and I believe it uses the standard format 🚀

Reply:
Updated, I had totally forgotten about that!